core/templates/dev/head/pages/story_editor/main_editor/StoryEditorDirective.js

```diff
@@ -15,6 +15,8 @@
 /**
  * @fileoverview Controller for the main story editor.
  */
+oppia.constant('EVENT_VIEW_NODE_EDITOR', 'viewNodeEditor');
```

Comment:
I think "story" should be somewhere in the name of both the event and the string constant. "Node" is very general -- it could refer to a graph node, for example.